pages/common/set.md

```diff
@@ -0,0 +1,19 @@
+# set
+
+> Display, set or unset value of shell attributes and positional parameters.
```

Comment:
Perhaps `value` would read better as `values` here?

Reply:
Yeah, that would sound better than `... value of shell attributes ...`, as the `shell attributes` is plural, so the `value` should also be plural.

Reply:
sure

Reply:
@sbrl If you could help me in squashing all the commits as a single commit.
Just like gerrit does.

Reply:
Yeah, when we merge a PR, we have a special button that lets us squash all the commits automagically - so you don't need to do anything there :slightly_smiling_face:

Reply:
ahhh ! thanks